packages/react-ui/src/components/ui/data-table/index.tsx

```diff
@@ -85,6 +86,7 @@ interface DataTableProps<
   ) => void;
   isLoading: boolean;
   filters?: F[];
+  FolderFilterComponenet?: React.ReactElement;
```

Comment:
nit: `folderFilterComponent`

I woudle event abstract it to be `customFilters: ReactNode[]`